CONTRIBUTING.md

```diff
@@ -9,12 +9,12 @@ Bun is written mostly in Zig, but WebKit & JavaScriptCore (the JavaScript engine
 Today (Feburary 2023), Bun's codebase has five distinct parts:
 
 - JavaScript, JSX, & TypeScript transpiler, module resolver, and related code
-- JavaScript runtime (`src/bun.js/`)
-- JavaScript runtime bindings (`src/bun.zig/bindings/**/*.cpp`)
-- Package manager (`src/install/`)
-- Shared utilities (`src/string_immutable.zig`)
+- JavaScript runtime ([`src/bun.js/`](src/bun.js/))
+- JavaScript runtime bindings ([`src/bun.zig/bindings/**/*.cpp`](src/bun.zig/bindings/))
```

Comment:
I should note that `src/bun.zig` isn't present in the repository, so I'm just going to assume that the build system generates it.

Reply:
src/bun.zig is definitely in the repo

 https://github.com/oven-sh/bun/blob/4c3879814289f79d81aef30e55c4bad9145c2af8/src/bun.zig#L33

Reply:
That's a file, the string implied it was a directory. I was wondering if it meant bun.js